.buildkite/pipeline.yml

```diff
@@ -0,0 +1,19 @@
+env:
+  NIX_PATH: "nixpkgs=channel:nixos-unstable"
+  CDN_DISTRIBUTION_ID: "E37J93462NMI5V"
+  CDN_BUCKET: "s3://ghc.dev-prod"
+  AWS_PROFILE: "ghc-dev"
+
+steps:
+  - commands:
+    - "nix-shell --run \"cabal v2-update && make\""
+    label: "make"
+    artifact_paths: "*.html"
+  - wait
+  - label: 'Deploy'
+    commands:
+    - "buildkite-agent artifact download '*.html'"
```

Comment:
:frowning_face: god forbid there is even a CSS file or something.

Reply:
Yeah, I would like it if there was an output dir